### PR TITLE
[APL-1179] fix CSV import/export

### PR DESCRIPTION
**Pull request by alzinchenko**
small refactoring;
adjusted escape/unescape procedure;
added the set of CTRL characters to escape/unescape methods;

### CODE REVIEW

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/files/shards/ShardsDownloadService.java [hunk 1]
```diff
@@ -50,6 +50,7 @@
     private final ShardNameHelper shardNameHelper = new ShardNameHelper();
     private final Map<Long, ShardDownloadStatus> shardDownloadStatuses = new HashMap<>();
     private static final int MIN_SHARDING_PEERS = 2;
+    public static final String FORCED_SHARD_ID_ENV = "APOLLO_FORCE_IMPORT_SHARD_ID";
```
Comment (silaev-firstbridge) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/files/shards/ShardsDownloadService.java]: do we need FORCED_SHARD_ID_ENV  to be public?

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/csv/CsvEscaperImpl.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+    public CsvEscaperImpl() {
+        this(DEFAULT_FIELD_DELIMITER);
+    }
+
+    public CsvEscaperImpl(char fieldDelimiter) {
+        if (fieldDelimiter != 0) {
+            ctrlCharacters[0] = fieldDelimiter;
+        }
+        this.translateMap = new HashMap<>(CTRL_CHARS_ESCAPE);
```
Comment (silaev-firstbridge) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/csv/CsvEscaperImpl.java]: Might me better to explisitly state //@NotThreadSafe at the top of this class?

File: apl-core/src/test/resources/logback-test.xml [hunk 3]
```diff
@@ -53,6 +53,9 @@
             <logger name="com.apollocurrency.aplwallet" level="debug" additivity="false">
                 <appender-ref ref="STDOUT" />
             </logger>
+            <logger name="com.apollocurrency.aplwallet.apl.core.shard.helper.csv.CsvReaderTest" level="trace" additivity="false">
```
Comment (AndrewBoyarsky) [inline apl-core/src/test/resources/logback-test.xml]: Pls ,Comment out  to avoid a lot of unnecessary  logs during build

Comment (alzinchenko) [reply apl-core/src/test/resources/logback-test.xml]: This logger is optional, and disabled by default. 
This change doesn't affect the number of logs during the build.

Comment (alzinchenko) [reply apl-core/src/test/resources/logback-test.xml]: Look through test-env.properties

**Pull request merged**